src/test/fuzz/txorphan.cpp

```diff
@@ -55,10 +55,18 @@ FUZZ_TARGET(txorphan, .init = initialize_orphanage)
             const auto num_out = fuzzed_data_provider.ConsumeIntegralInRange<uint32_t>(1, 256);
             // pick outpoints from outpoints as input. We allow input duplicates on purpose, given we are not
             // running any transaction validation logic before adding transactions to the orphanage
-            for (uint32_t i = 0; i < num_in; i++) {
-                auto& prevout = PickValue(fuzzed_data_provider, outpoints);
-                // try making transactions unique by setting a random nSequence, but allow duplicate transactions if they happen
-                tx_mut.vin.emplace_back(prevout, CScript{}, fuzzed_data_provider.ConsumeIntegralInRange<uint32_t>(0, CTxIn::SEQUENCE_FINAL));
+            {
+                // Create copy of set to allow fast PickValue
```

Comment:
Perhaps a more descriptive comment would be good, e.g.: `We don't call PickValue on the outpoints set directly because PickValue will advance a begin iterator on the set, which is expensive, because it only has a "++" operator. As it would be called in a loop of num_in (~outpoints.size()), the runtime would be O(outpoints.size() ^ 2). Instead we create a std::vector copy of the set, which allows for O(1) std::advance.`.